xtext/Dockerfile

```diff
@@ -43,6 +47,7 @@ RUN mvn org.apache.maven.plugins:maven-dependency-plugin:3.6.0:get -Dartifact=co
 
 FROM tomcat:9.0.76-jdk17-temurin
 
+ARG ES_ADDRESS
```

Comment:
What is this for?

Reply:
URL of Xtext endpoint. It is documented in [here](https://github.com/mdenet/educationplatform-docker/blob/development_server/README.md#environment-variables)

Reply:
```suggestion
# See https://github.com/mdenet/educationplatform-docker/blob/development_server/README.md#environment-variables
ARG ES_ADDRESS
```